app/src/main/res/layout/fragment_manual_log.xml

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
layout breaks when user opens the keyboard 
![Screenshot 2024-03-20 152006](https://github.com/Starlight-Solutions/android-main/assets/71108052/620415e5-2718-41ba-95d0-9d8da0f1de00)